.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -5,7 +5,7 @@ about: Report a bug
 ---
 
 <!--
-Hello! Thanks for contributing. 
+Hello! Thanks for contributing.  In order to help us make address this issue quickly and effectively, please:

```

Comment:
`s/make /`, or "For the fastest response and resolution, please:"

Reply:
Oh yeah, I like that better